packages/@react-aria/combobox/test/useComboBox.test.js

```diff
@@ -132,6 +132,18 @@ describe('useComboBox', function () {
     expect(toggleSpy).toHaveBeenLastCalledWith(null, 'manual');
   });
 
+  it('should not prevent onBlur when no button provided', function () {
```

Comment:
generally like to name it for what it does, not for the bug it was fixing
```suggestion
  it('should call onBlur when no button provided and you leave the field', function () {
```

Reply:
good to know, thank you ;)